src/Build/Resources/Strings.resx

```diff
@@ -478,6 +478,10 @@
     <value>MSB4110: Expected a property at position {1} in condition "{0}". Did you forget the opening parenthesis after the '$'? To use a literal '$', use '%24' instead.</value>
     <comment>{StrBegin="MSB4110: "}</comment>
   </data>
+  <data name="IllFormedPropertyWhitespaceInCondition" xml:space="preserve">
+    <value>MSB4088: Condition "{0}" is improperly constructed with whitespace. Remove whitespaces from the property.</value>
```

Comment:
Why does this error code not match the one below? Also, both of these are taken.
```suggestion
    <value>MSBxxxx: Unexpected whitespace at position "{1}" of condition "{0}".</value>
```

Reply:
Fixed, thanks